docs/tutorial/04_integrating_magma_access_gateway_with_magma_orchestrator.md

```diff
@@ -0,0 +1,81 @@
+# 4. Integrating Magma Access Gateway with Magma Orchestrator 
+
+## Integrate Magma Access Gateway with Magma Orchestrator
+
+Offer an application endpoint from Orchestrator:
+
+```console
+juju offer orc8r.orc8r-nginx:orchestrator
+juju consume orc8r.orc8r-nginx
+```
+
+Relate Magma Access Gateway with Orchestrator:
+
+```console
+juju relate orc8r-nginx:orchestrator magma-access-gateway-operator
+```
+
+Wait for the application to go back to `Active-Idle`:
+
+```console
+
+ubuntu@host:~$ juju status
+Model  Controller     Cloud/Region   Version  SLA          Timestamp
+edge   aws-us-east-2  aws/us-east-2  2.9.42   unsupported  16:09:01-05:00
+
+App                            Version  Status  Scale  Charm                          Channel  Rev  Exposed  Message
+magma-access-gateway-operator           active      1  magma-access-gateway-operator  stable    29  no       
+
+Unit                              Workload  Agent  Machine  Public address  Ports  Message
+magma-access-gateway-operator/0*  active    idle   0        18.189.227.182         
+
+Machine  State    Address         Inst id                Series  AZ  Message
+0        started  18.189.227.182  manual:18.189.227.182  focal       Manually provisioned machine
+```
+
+Fetch the Access Gateway's `Hardware ID` and `Challenge Key` and note those values:
+
+```console
+juju run-action magma-access-gateway-operator/0 get-access-gateway-secrets --wait
+```
+
+## Create a network in Magma Orchestrator
+
+### Create a user in the `magma-test` organization:
+
+1. Login to the `master` Orchestrator organization at this address: `https://master.nms.<your domain>`.
+2. Click on the :fontawesome-solid-user-plus: icon next to the `magma-test` organization
+3. Add a user with the following attributes:
+    * email: `admin@juju.com`
+    * password: `password123`
+    * role: `Super User`
+
+### Create a network in the `magma-test` organization:
+
+1. Login to the `magma-test` organization at this address: `https://magma-test.nms.<your domain>`.
+2. On the left pane, click on "Networks"
+3. Click on "Add Network"
+4. Fill in the following values:
+    * Network ID: `my-network`
+    * Network Name: `my-network`
+    * Description: `my-network`
+    * Network Type: `LTE`
+5. Refresh the page. You should now see your network dashboard
+
+### Add the Access Gateway to the network
+
+1. Navigate to "Equipment" via the left pane
+2. Click on "Add New"
+3. Fill in the following values:
+    * Gateway Name: `my-gateway`
+    * Gateway ID: `my-gateway`
+    * Hardware UUID: `<Hardware ID from the previous step>`
+    * Gateway Description: `my-gateway`
+    * Challenge Key: `<Challenge Key from the previous step>`
+4. Click on "Save and Continue"
```

Comment:
Isn't there more tabs, like the `EPC` section? I find this section can make or break setup.

Reply:
The new wording is better, thanks!